doc/cephfs/cephfs-mirroring.rst

```diff
@@ -398,6 +398,39 @@ created, the mirror daemon will clear the failed state upon successful synchroni
 When mirroring is disabled, the respective `fs mirror status` command for the file system
 will not show up in command help.
 
+Exported Metrics
```

Comment:
@joscollin please include a section about labeled perf counters with some explanation before detailing out the counters themselves.